app/retail/views.py

```diff
@@ -26,7 +26,14 @@
 
 
 def index(request):
+    slides = [
+        ("Kevin Owocki","/static/v2/images/avatar.png", "This could be the next freelance boom."),
+        ("Aditya Anand","/static/v2/images/avatar.png", "I feel it is so awesome to have the opportunity through Gitcoin to do what I love and get paid for it, and to have reasonable freedom about the way I work, that it already seems too good to be true."),
+        ("Kevin Owocki","/static/v2/images/avatar.png", "Staying lean, building the community, iterating... The way to go vs many projects that are just looking for the money grab immediately.")
+    ]
     context = {
+        'slides': slides,
+        'slideDuration': 6000,
```

Comment:
# ms

Reply:
Yup! we could rename it to `slideDurationInMs` or leave a comment -> take your pick

Reply:
up to you